### ISSUE TITLE
Linux "Could not find executable"

### ISSUE THREAD

**Issue by AdminRAT** [discordconsole-team/DConsoleLauncher#1]
Whenever launching DiscordConsole from linux, it could not find the executable. But I have to do ```./DiscordConsole``` when it's looking for the executable.

```
Could not execute DiscordConsole
Executable name: ./DiscordConsole
DiscordConsole 2.3.9
...

**Comment by jD91mZM2** [comment]
Seems like I missed that when developing it. I think I was still on Windows back then.  
@Mnpn03 Seems easy enough. Just add `./` before the default executable thing.  
Just hope it still works on Windows :^)

**Comment by AdminRAT** [comment]
Windows still works. Just linux needs that ```./```

**Comment by jD91mZM2** [comment]
Oh no, it's using the same command for both currently.

**Comment by AdminRAT** [comment]
That makes sense, seeming when it works on Windows, and not on Linux.

**Comment by jD91mZM2** [comment]
Yeah, I guess I just assumed it would work on both without trying.
